### PR TITLE
Remove unnecessary extra </div>

### PR DESCRIPTION
**Pull request by masatake**

### CODE REVIEW

File: source/documentation/admin-guide/administration-guide.html.md [hunk 1]
```diff
@@ -1170,7 +1170,7 @@ This applies to all services, but you should be especially aware of the hazards
 *   Directory Services
 *   DNS
 *   Storage
-    </div>
```
Comment (sandrobonazzola) [inline source/documentation/admin-guide/administration-guide.html.md]: Isn't this div closing the div opened at line 1165?  <div class="alert alert-info">

**Comment by sandrobonazzola** [comment]
Inline comment

**Comment by masatake** [comment]
Look at http://www.ovirt.org/documentation/admin-guide/administration-guide/.
The bottom half of the page rendered in "blue". I don't know well about html and the page structure.
However, `</div>` is at the head of area rendered in "blue".
div

This is the expected rendering result? If yes, I have to withdraw this PR.

**Comment by masatake** [comment]
I recongnied removing `</div>` is not good.
`## Logical Networks` is also strange.
Should I convert this to an issue instead of pull request?

**Comment by sandrobonazzola** [comment]
This second change make more sense to me, Need to check the real output since preview here is not really helping

**Comment by sandrobonazzola** [comment]
@thatdocslady can you review this patch?

**Comment by sandrobonazzola** [comment]
Obsoleted by pull  #140

**Pull request closed without merging**